extensions/sql-database-projects/package.json

```diff
@@ -524,7 +541,45 @@
           "when": "connectionProvider == 'MSSQL' && mssql:engineedition != 11"
         }
       ]
-    }
+    },
+    "problemMatchers": [
+      {
+				"name": "sqlproj-problem-matcher",
+				"owner": "sqlproj-builder",
+				"fileLocation": "absolute",
+				"pattern": {
+					"regexp": "^(.*\\.sql)\\((\\d+),(\\d+),(\\d+),(\\d+)\\):\\s+\\w+\\s+(warning|error)\\s+(\\w+):\\s+(.*?)?$",
```

Comment:
Can you give an example of a match looks like?

Reply:
Example error match will be like this

Table1.sql(12,5,12,20): error KE : Syntax is invalid